packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js

```diff
@@ -2,6 +2,16 @@ import asyncStates from '@zooniverse/async-states'
 import { autorun } from 'mobx'
 import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
 
+const NO_IMAGE_TOOLBAR_MIME_TYPES = [
```

Comment:
A shorter affirmative list would be `IMAGE_TOOLBAR_MIME_TYPES`. Refactoring the `showImageToolbar` view accordingly might read more easily.